.github/workflows/build-test-package.yml

```diff
@@ -61,12 +61,12 @@ jobs:
         ninja
 
     - name: Build ITK
-      if: matrix.os == 'windows-2019'
+      if: matrix.os == 'windows-2022'
       run: |
         cd ..
         mkdir ITK-build
         cd ITK-build
-        call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
+        call "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
```

Comment:
I tried going to 2022 in https://github.com/dzenanz/ITKIOOMEZarrNGFF/tree/tensorstore, but after realizing it is not working I reverted back to 2019. Now I realize that rename `2019`->`2022` is not the only thing here, but change from `Program Files (x86)` to `Program Files` (64-bit).

Reply:
Note that the bump for v5.3rc04.post4 contains a couple of other small changes as well, see the commit history at https://github.com/InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/pull/9/commits